test/test_exon_object.py

```diff
@@ -25,8 +27,8 @@ def test_exon_object_by_id():
 
 def test_exon_object_by_id_on_negative_strand():
     """
-    test_exon_object_by_id : check properties of exon 1 from CXCR3 when looked
-    up by ID in Ensembl 77.
+    test_exon_object_by_id_on_negative_strand : check properties of exon 1
```

Comment:
It's to combat something annoying that nose does when you're running tests. If a test has a docstring then the name nose shows gets replaced by the docstring, which then obscures which test it's running.